traffic_ops/app/db/trafficvault/migrations/2021042200000000_init.up.sql

```diff
@@ -1,16 +0,0 @@
-/*
```

Comment:
Now that the `db/trafficvault/migrations` directory is empty, `git` does not create it. This breaks our db tests:

> ```shell
> ./.github/actions/todb-tests/entrypoint.sh: line 35: pushd: traffic_ops/app/db/trafficvault/migrations: No such file or directory
> ```



Maybe add a `.gitkeep` or `.gitignore` file to ensure the directory exists?

Reply:
It actually breaks if there are no migrations in that directory. Which may explain why the init up and down migrations for Traffic Vault were just
```sql
SELECT;
```
I think it would be better to fix the test to not worry about the case that there are no migrations

Reply:
Looking at `trafficops.log` within the `ciab-logs.zip` artifact again, migrations fail at 992d4a0a54 due to the directory not existing:
```
INFO:root:Database load_schema succeeded
DEBUG:root:admin exec failed; stderr: Starting Migrate: open /opt/traffic_ops/app/db/trafficvault/migrations/.: no such file or directory

	stdout: 
ERROR:root:Failed to set up database: Database migrate failed
ERROR:root:Can't connect to the database.  Use the script `/opt/traffic_ops/install/bin/todb_bootstrap.sh` on the db server to create it and run `postinstall` again.
Error on line 38 of /opt/traffic_ops/install/bin/postinstall
```